engine/src/flutter/lib/web_ui/test/engine/semantics/semantics_test.dart

```diff
@@ -4397,6 +4397,31 @@ void _testLink() {
     expect(object.element.tagName.toLowerCase(), 'a');
     expect(object.element.getAttribute('href'), 'https://flutter.dev');
   });
+
+  test('a node that is both a link and a button is rendered as a link', () {
```

Comment:
I still need to get this fixed

Reply:
Fixed in https://github.com/flutter/flutter/pull/174473/commits/17ffb0c42522fcc0e3e9d12d279e943d992994e7